modules/living_spaces_group/src/LivingSpacesGroupManager.php

```diff
@@ -44,4 +44,24 @@ public function getLivingSpaceGroupTypes() {
     return is_array($types) ? $types : [];
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getEntityTypesOfLivingSpaceGroupTypes() {
+    $types = $this->getLivingSpaceGroupTypes();
+    $entity_types = [];
+    foreach ($types as $type) {
+      /** @var \Drupal\group\Entity\GroupContentType[] $group_content_types */
+      $group_content_types = \Drupal::entityTypeManager()
```

Comment:
Please, use `$this->entityTypeManager` instead of `\Drupal::entityTypeManager()`.